src/coreclr/vm/arm64/stubs.cpp

```diff
@@ -252,11 +252,11 @@ class LoadFromLabelInstructionFormat : public InstructionFormat
             _ASSERTE((variationCode & 0x1F) != 31);
 
             // adrp Xt, #Page_of_fixedUpReference
-            *((DWORD*)pOutBuffer) = ((9<<28) | ((imm & 3)<<29) | (imm>>2)<<5 | (variationCode&0x1F));
+            *((DWORD*)pOutBufferRW) = ((9<<28) | ((imm & 3)<<29) | (imm>>2)<<5 | (variationCode&0x1F));
```

Comment:
```suggestion
            *((DWORD*)pOutBufferRW) = ((9<<28) | ((imm & 3)<<29) | (imm>>2)<<5 | (variationCode & 0x1F));
```
better to read and consistent with the line below.

Reply:
I was trying to not to modify anything else than necessary to eliminate noise for the review. There are so many places in the runtime where spacing around operators is inconsistent that it seems that if we wanted to tackle that, a separate PR with sweeping search and replace over the whole codebase would be a better way to tackle it.

Reply:
sounds good.